io.c

```diff
@@ -10122,15 +10112,24 @@ rb_io_advise(int argc, VALUE *argv, VALUE io)
 
 /*
  *  call-seq:
- *     IO.select(read_array [, write_array [, error_array [, timeout]]]) -> array or nil
+ *    IO.select(read_ios, write_ios = [], error_ios = [], timeout = nil) -> array or nil
+ *
+ *  Calls system call {select(2)}[https://linux.die.net/man/2/select].
```

Comment:
1. Purely aesthetical: "Calls system call" reads a bit odd. Maybe "Invokes system call" or something like that?
2. I believe we should leave "what it does" phrase the very next after the first, e.g. "Invokes system call `select`, which monitors given \IO objects..." — so it would be immediately obvious _what and why_ this method does (e.g. answer "do I need it? if so, why?") and only then the details of arguments and return values.